api-documentation.js

```diff
@@ -131,9 +131,13 @@ class ApiDocumentation extends EventsTargetMixin(AmfHelperMixin(LitElement)) {
   }
 
   _summaryTemplate() {
-    const { _docsModel, baseUri } = this;
+    const { _docsModel, baseUri, rearrangeEndpoints } = this;
 
-    return html`<api-summary .amf="${_docsModel}" .baseUri="${baseUri}"></api-summary>`;
+    return html`<api-summary
+        .amf="${_docsModel}"
+        .baseUri="${baseUri}"
+        .rearrangeendpoints="${rearrangeEndpoints}"
```

Comment:
This is a boolean attribute! Do not use dot in front of it in the template. It will set a value of `"undefined"` (as string) is missing. Also when using passing as a property notation then use camel case values: `.rearrangeEndpoints="${rearrangeEndpoints}"`.